tests/subsys/fs/littlefs/src/test_lfs_basic.c

```diff
@@ -80,8 +82,7 @@ static int create_write_hello(const struct fs_mount_t *mp)
 	TC_PRINT("creating and writing file\n");
 
 	zassert_equal(fs_open(&file,
-			      testfs_path_init(&path, mp,
-					       HELLO,
+			      testfs_path_init(&path, mp, HELLO,
```

Comment:
Please avoid gratuitous reformatting until we have a tool that does it.  The line separation here was intentional.

Also: there is no change in this file other that the reformatting and introduction of two macros that aren't used.

Reply:
Removed